src/Processors/Formats/Impl/LineAsStringRowInputFormat.cpp

```diff
@@ -0,0 +1,107 @@
+#include <Processors/Formats/Impl/LineAsStringRowInputFormat.h>
+#include <Formats/JSONEachRowUtils.h>
+#include <common/find_symbols.h>
+#include <IO/ReadHelpers.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int LOGICAL_ERROR;
+    extern const int INCORRECT_DATA;
+}
+
+LineAsStringRowInputFormat::LineAsStringRowInputFormat(const Block & header_, ReadBuffer & in_, Params params_) :
+    IRowInputFormat(header_, in_, std::move(params_)), buf(in)
+{
+    if (header_.columns() > 1 || header_.getDataTypes()[0]->getTypeId() != TypeIndex::String)
+    {
+        throw Exception("This input format is only suitable for tables with a single column of type String.", ErrorCodes::LOGICAL_ERROR);
+    }
+}
+
+void LineAsStringRowInputFormat::resetParser()
+{
+    IRowInputFormat::resetParser();
+    buf.reset();
+}
+
+void LineAsStringRowInputFormat::readLineObject(IColumn & column)
+{
+    PeekableReadBufferCheckpoint checkpoint{buf};
+    size_t balance = 0;
+
+    if (*buf.position() == ';') {
+        ++buf.position();
+        if(buf.eof())
+            return;
+    }
+
+    if (*buf.position() != '"')
+        throw Exception("Line object must begin with '\"'.", ErrorCodes::INCORRECT_DATA);
```

Comment:
why? It can be any new-line separated data.

Reply:
I hypothesis the line is quoted by ".
Please give me some insert query and  select result that this feature should convered.(For example, another new-line separated queries). So that I can understand better. Thank you

Reply:
```
cat /proc/cpuinfo | clickhouse-client -q 'insert into table format LineAsString' 
cat /var/log/dmesg | clickhouse-client -q 'insert into table format LineAsString' 
```

i.e. whatever data. The only assumption is: the row ends when you see newline character (even PeekableReadBuffer may be an overkill :) )

Reply:
@filimonov 


```
cat /proc/cpuinfo
   1 processor       : 0
   2 vendor_id       : GenuineIntel
   3 cpu family      : 6
   4 model           : 79
   5 model name      : Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz
```

after execute
```
cat /proc/cpuinfo | clickhouse-client -q 'insert into table format LineAsString' 
```

the result we expected returns 5 rows?
```
select * from line_as_string:
|--field-----------|
|processor       : 0|
|vendor_id       : GenuineIntel|
|cpu family      : 6|
|model           : 79|
|model name      : Intel(R) Xeon(R) CPU E5-2640 v4 @ 2.40GHz|

5 rows returned?

```

Reply:
Yes, and if there is an empty line at the end - then 6. 
(of course, don't base the real test on cpuinfo - it differs on every machine)

Reply:
OK, I will fix this.

Reply:
@filimonov  Is the new commit matched what we need?